### ISSUE TITLE
ModuleNotFoundError in tftservices

### ISSUE THREAD

**Issue by robvanmieghem** [threefoldfoundation/tft-stellar#477]
tftservices version 1.5.0-rc1

```log
2023-03-15 14:21:12.533 | ERROR    | jumpscale.tools.errorhandler.errorhandler:_handle_exception:93 - ModuleNotFoundError: No module named 'tfchaintypes'
Traceback (most recent call last):
> File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/bin/threebot", line 8, in <module>
    sys.exit(cli())
    │   │    └ <Group cli>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f03639289d0>
           └ <Group cli>
  File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/lib/python3.8/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f03660ebee0>
         │    └ <function MultiCommand.invoke at 0x7f036392d9d0>
         └ <Group cli>
  File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f0366246af0>
           │               │       │       └ <function Command.invoke at 0x7f036392d4c0>
           │               │       └ <Command start>
           │               └ <click.core.Context object at 0x7f0366246af0>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f03662a61f0>
  File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'development': True, 'identity': None, 'domain': None, 'email': None, 'background': False, 'local': False, 'cert': True}
           │   │      │    │           └ <click.core.Context object at 0x7f0366246af0>
           │   │      │    └ <function start at 0x7f0361abcb80>
           │   │      └ <Command start>
           │   └ <function Context.invoke at 0x7f0363928280>
           └ <click.core.Context object at 0x7f0366246af0>
  File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/lib/python3.8/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'development': True, 'identity': None, 'domain': None, 'email': None, 'background': False, 'local': False, 'cert': True}
                       └ ()
  File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/lib/python3.8/site-packages/jumpscale/entry_points/threebot.py", line 155, in start
    j.servers.threebot.start_default(wait=True, local=local, domain=domain, email=email, cert=cert)
    │ │                                         │     │      │      │       │     │      │    └ True
    │ │                                         │     │      │      │       │     │      └ True
    │ │                                         │     │      │      │       │     └ None
    │ │                                         │     │      │      │       └ None
    │ │                                         │     │      │      └ None
    │ │                                         │     │      └ None
    │ │                                         │     └ False
    │ │                                         └ False
    │ └ <property object at 0x7f0363972f40>
    └ <jumpscale.loader.J object at 0x7f0363930e50>
  File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/lib/python3.8/site-packages/jumpscale/servers/threebot/__init__.py", line 34, in start_default
    server.start(wait=wait, cert=cert)
    │      │     │    │     │    └ True
    │      │     │    │     └ True
    │      │     │    └ True
    │      │     └ True
    │      └ <function ThreebotServer.start at 0x7f0362c6cee0>
    └ ThreebotServer(
        instance_name='default',
        _package_manager=StoredFactory(PackageManager),
        domain=None,
        email=None,
        a...
  File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/lib/python3.8/site-packages/jumpscale/servers/threebot/threebot.py", line 882, in start
    self.packages._install_all()
    │    └ <property object at 0x7f0362c68c70>
    └ ThreebotServer(
        instance_name='default',
        _package_manager=StoredFactory(PackageManager),
        domain=None,
        email=None,
        a...
  File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/lib/python3.8/site-packages/jumpscale/servers/threebot/threebot.py", line 701, in _install_all
    self.install(pkg)
    │    │       └ <jumpscale.servers.threebot.threebot.Package object at 0x7f035ba7eee0>
    │    └ <function PackageManager.install at 0x7f0362c6c3a0>
    └ PackageManager(
        instance_name='default',
        parent=ThreebotServer(instance_name_='default'),
        packages={'auth': {'path': '/...
  File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/lib/python3.8/site-packages/jumpscale/servers/threebot/threebot.py", line 642, in install
    self.threebot.gedis._system_actor.register_actor(actor["name"], actor["path"], force_reload=True)
    │    │                                           │              └ {'name': 'tfchainmigration_service_tfchainmigration_service', 'path': '/code/threefoldfoundation/tft-stellar/ThreeBotPackages...
    │    │                                           └ {'name': 'tfchainmigration_service_tfchainmigration_service', 'path': '/code/threefoldfoundation/tft-stellar/ThreeBotPackages...
    │    └ <property object at 0x7f0362c68860>
    └ PackageManager(
        instance_name='default',
        parent=ThreebotServer(instance_name_='default'),
        packages={'auth': {'path': '/...
  File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/lib/python3.8/site-packages/jumpscale/servers/gedis/baseactor.py", line 25, in wrapper
    result = func(*bound.args, **bound.kwargs)
             │     │     │       │     └ <property object at 0x7f0364e06310>
             │     │     │       └ <BoundArguments (self=<jumpscale.servers.gedis.systemactor.SystemActor object at 0x7f0362c80730>, actor_name='tfchainmigratio...
             │     │     └ <property object at 0x7f0364e062c0>
             │     └ <BoundArguments (self=<jumpscale.servers.gedis.systemactor.SystemActor object at 0x7f0362c80730>, actor_name='tfchainmigratio...
             └ <function SystemActor.register_actor at 0x7f0362c788b0>
  File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/lib/python3.8/site-packages/jumpscale/servers/gedis/systemactor.py", line 53, in register_actor
    module = j.tools.codeloader.load_python_module(actor_path, force_reload=force_reload)
             │ │                                   │           │            └ True
             │ │                                   │           └ True
             │ │                                   └ '/code/threefoldfoundation/tft-stellar/ThreeBotPackages/tfchainmigration_service/actors/tfchainmigration_service.py'
             │ └ <property object at 0x7f0363972ea0>
             └ <jumpscale.loader.J object at 0x7f0363930e50>
  File "/root/.cache/pypoetry/virtualenvs/tft-stellar-Z1Rq3U3B-py3.8/lib/python3.8/site-packages/jumpscale/tools/codeloader/__init__.py", line 48, in load_python_module
    spec.loader.exec_module(module)
    │    │      │           └ <module 'tfchainmigration_service' from '/code/threefoldfoundation/tft-stellar/ThreeBotPackages/tfchainmigration_service/acto...
    │    │      └ <function _LoaderBasics.exec_module at 0x7f03662a0670>
    │    └ <_frozen_importlib_external.SourceFileLoader object at 0x7f0360799c40>
    └ ModuleSpec(name='tfchainmigration_service', loader=<_frozen_importlib_external.SourceFileLoader object at 0x7f0360799c40>, or...
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "/code/threefoldfoundation/tft-stellar/ThreeBotPackages/tfchainmigration_service/actors/tfchainmigration_service.py", line 26, in <module>
    from tfchaintypes.CryptoTypes import PublicKey, PublicKeySpecifier

ModuleNotFoundError: No module named 'tfchaintypes'

```

**Comment by robvanmieghem** [comment]
79c969214484be1995bc0415a79be9454f86bdfe

**Comment by robvanmieghem** [comment]
published version 1.5.0-rc2
